src/pages-and-resources/discussions/app-config-form/apps/shared/discussion-restrictions/RestrictDatesInput.jsx

```diff
@@ -72,13 +74,15 @@ RestictDatesInput.propTypes = {
   fieldClasses: PropTypes.string,
   formGroupClasses: PropTypes.string,
   fieldNameCommonBase: PropTypes.string.isRequired,
+  dataTestId: PropTypes.string,
 };
 
 RestictDatesInput.defaultProps = {
   fieldClasses: '',
   helpText: '',
   feedbackClasses: '',
   formGroupClasses: '',
+  dataTestId: '',
```

Comment:
It would be better not to have empty default value.

Reply:
Ok. 
**Best Practise**: Instead of using an empty string, we can set the default value of dataTestId to null. This indicates that the prop is intentionally not provided and avoids adding potentially confusing or meaningless test IDs to our components.